### ISSUE TITLE
News API No longer working

### ISSUE THREAD

**Issue by tadder** [Qrome/marquee-scroller#176]
I have been using the Reuters newsfeed for sometime with no problems but in the last few months it has stoped updating. It now tells me to configure news, but there is nothing in the drop down list to select. 
Anyone any ideas?

**Comment by Qrome** [comment]
Yeah, it sounds like you are running an older version.   The news API changed over a year a go to where they only allowed HTTPS to grab the list of news feeds.   The list is now hosted here on github.   The latest code will reference it.

**Comment by Qrome** [comment]
Link to release that included this fix.   https://github.com/Qrome/marquee-scroller/releases/tag/Scroller_V2.16
